ci/drop-canton.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+# Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+set -xeuo pipefail
```

Comment:
Generally speaking we don't want `-x` on CI as that risks exposing credentials in various ways.